src/components/Table/index.js

```diff
@@ -1,21 +1,73 @@
 // import {useState} from 'react';
-import StyledRow from './StyledRow';
-import StyledRowItems from './StyledRowItems';
+import {nanoid} from 'nanoid';
+import {useState} from 'react';
+import Styled from 'styled-components';
```

Comment:
I think "styled" is normally written lowercase.

Reply:
Possibly. I'm writing it this way, because this is how  CJ showed it to us.